Add note about user/system python package installation (linux) in the… (#3194)
ros2/ros2_documentation: source/The-ROS2-Project/Contributing/Contributing-To-ROS-2-Documentation.rst

  .. group-tab:: Linux

>     Either do a user-specific install (recommended, requires ``~/.local/bin/`` to be added to ``$PATH``):

Reviewer: ```suggestion
    The next command does a user-specific install, which requires ``~/.local/bin/`` to be added to ``$PATH``:
```

    .. code-block:: console

       pip3 install --user --upgrade -r requirements.txt

    Or a system install (requires superuser access):

    .. code-block:: console

       pip3 install --upgrade -r requirements.txt

> 

Reviewer: I don't think we should recommend this.  In most cases, the Linux distributions have patched it so that this does a user install anyway.  And I think that pip discourages this with a warning (though I don't have a setup I can try this in right now).

Author: Jus tested this on my (relatively) fresh Ubuntu 22.04 LTS install. Pip doesn't warn about this. However, if that's the case then I agree with you, we should probably not recommend this.
